### ISSUE TITLE
High-Resolution Image Synthesis with Latent Diffusion Models

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#185]
# TL; DR

- Pretrain an autoencoder with smooth latent geometry; execute diffusion process in the latent space.

# Motivation

- High resolution diffusion models are extremely compute-hungry.
- Likelihood models devote so much resources on imperceptible details (perceptual compression)

![image](https://user-images.githubusercontent.com/46308407/188567007-aa770f65-8da6-4a9e-b75a-10bf80a82bf7.png)

- With the help of an autoencoder, we can remove these (imperceptible) high frequency details and focus on semantic (canonical) visual features.

# Formulation

![image](https://user-images.githubusercontent.com/46308407/188567159-9fe4a1ab-c60f-408e-8c97-9bfb8ee426a4.png)

- Train an autoencoder with either KL regularization or VQ (Vector Quantization).
- This is to learn a smooth, well-behaved latent space.

- Provide conditioning information through **cross-attention** w.r.t. the bottleneck feature.
- Employ a domain expert, either a language model, semantic map model or generic image encoder.
- CLIP L/14 text encoder (frozen) is used for later version of stable diffusion, following Imagen #180 .

# Evaluation

![image](https://user-images.githubusercontent.com/46308407/188567588-dfb53818-3e78-46b5-be53-255436188b77.png)
![image](https://user-images.githubusercontent.com/46308407/188567632-177bc2e7-45c7-46b8-9202-fc3c0de7eca4.png)

- Text-to-Image: 1.45B parameter diffusion model (LDM-8-KL) trained with LAION-400M.

- Compression rate analysis.
![image](https://user-images.githubusercontent.com/46308407/188567888-b02cd41e-d300-4135-915d-bd5bca8ff261.png)

- Quantitative comparisons.
![image](https://user-images.githubusercontent.com/46308407/188567924-d91cf1c2-2e00-48b6-9936-d98ddba65fab.png)
![image](https://user-images.githubusercontent.com/46308407/188568080-9e00b907-d8d9-464d-a68c-c5573af4cd0f.png)

- Diverse scenarios.
![image](https://user-images.githubusercontent.com/46308407/188567954-e079ccd5-f674-4cdb-ac21-f260198e9655.png)
![image](https://user-images.githubusercontent.com/46308407/188567974-8e1e2da9-61ac-4e86-8e37-46b3aab18223.png)
![image](https://user-images.githubusercontent.com/46308407/188568004-0600bf96-f6d8-4cdf-a226-47c35e85f303.png)
![image](https://user-images.githubusercontent.com/46308407/188568172-b61209a1-1f9a-4746-b9cf-49203a84fa93.png)

![image](https://user-images.githubusercontent.com/46308407/188568115-62fc519e-fb64-4a32-8350-843f356800dd.png)
![image](https://user-images.githubusercontent.com/46308407/188568134-59e016f2-41b5-4102-b5e7-b28e3abc0e7c.png)
![image](https://user-images.githubusercontent.com/46308407/188568156-72110005-8527-4232-a7d3-6c537319c901.png)

# Inspiration

- Rather simple formulation that affords enormous opportunities to the research community by *democratizing high-resolution image diffusion*.
- Their code base and pretrained weights (stable diffusion) really bring much research impact.
